src/mlpack/core/optimizers/cmaes/cmaes.hpp

```diff
@@ -0,0 +1,252 @@
+/**
+ * @file cmaes.hpp
+ * @author Marcus Edel
+ * @author Kartik Nighania
+ *
+ * Covariance Matrix Adaptation Evolution Strategy
+ *
+ * mlpack is free software; you may redistribute it and/or modify it under the
+ * terms of the 3-clause BSD license.  You should have received a copy of the
+ * 3-clause BSD license along with mlpack.  If not, see
+ * http://www.opensource.org/licenses/BSD-3-Clause for more information.
+ */
+#ifndef MLPACK_CORE_OPTIMIZERS_CMAES_CMAES_HPP
+#define MLPACK_CORE_OPTIMIZERS_CMAES_CMAES_HPP
+
+namespace mlpack {
+namespace optimization {
+
+/**
+*CMA-ES stands for Covariance Matrix Adaptation Evolution Strategy.
```

Comment:
Can you use a space between ``*`` and the beginning of the comment and also align the second ``*``with the first?

```
/**
 * CMA-ES stands for Covariance Matrix Adaptation Evolution Strategy.
 * Evolution strategies (ES) are stochastic, derivative-free methods for 
```